test/clj/chronograph/handlers/organization_test.clj

```diff
@@ -93,3 +93,49 @@
         (is (s/valid? :organizations/organization
                       (:body response))
             "Slug length can be exactly 256 characters at most.")))))
+
+
+(deftest find-one-organization
+  (testing "when fetching an organization"
+    (let [user (factories/create-user)
+          other-user (factories/create-user)
+          slug "bar"
+          slug-that-doesnt-exist (str (java.util.UUID/randomUUID))
+          organization (:body (create-organization
+                               {:body {:name "foo" :slug slug}
+                                :user user}))]
+      (is (= {:status 200
+              :headers {}
+              :body organization}
+             (organization/find-one {:params {:slug slug}
+                                     :user user}))
+          "Fetching an organization with authorized user returns the organization details.")
+      (is (= {:status 404
+              :headers {}
+              :body {:error "Not found"}}
+             (organization/find-one {:params {:slug slug}
+                                     :user other-user}))
+          "Fetching an organization with unauthorized user fails with HTTP error.")
+      (is (= {:status 400
+              :headers {}
+              :body {:error "Bad slug"}}
+             (organization/find-one {:params {:slug nil}
+                                     :user user}))
+          "Fetching an organization with invalid slug fails with HTTP error.")
+      (is (= {:status 404
+              :headers {}
+              :body {:error "Not found"}}
+             (organization/find-one {:params {:slug slug-that-doesnt-exist}
+                                     :user user}))
+          "Fetching a non-existing organization fails with HTTP error.")
+      (is (= {:status 404
+              :headers {}
+              :body {:error "Not found"}}
+             (do ;; create org so the "other-user" gets registered in the ACL
```

Comment:
Put this `do` and `create-organization` outside the `is` sexp.

Reply:
Hm, OK. I thought it reads better this way.

Reply:
Leaving it as-is because I think it does in fact become less readable, and also it turns out that the `do` block is considered spurious outside the `is` form (clj-kondo complains).